core/vm/gas_table.go

```diff
@@ -403,25 +403,6 @@ func gasCall(evm *EVM, contract *Contract, stack *Stack, mem *Memory, memorySize
 		return 0, ErrGasUintOverflow
 	}
 
-	if evm.chainRules.IsEIP4762 {
-		// If value is transferred, it is charged before 1/64th
-		// is subtracted from the available gas pool.
-		if transfersValue {
-			gas, overflow = math.SafeAdd(gas, evm.Accesses.TouchAndChargeValueTransfer(contract.Address().Bytes()[:], address.Bytes()[:]))
-			if overflow {
-				return 0, ErrGasUintOverflow
-			}
-		}
-	}
-
-	evm.callGasTemp, err = callGas(evm.chainRules.IsEIP150, contract.Gas, gas, stack.Back(0))
-	if err != nil {
-		return 0, err
-	}
-	if gas, overflow = math.SafeAdd(gas, evm.callGasTemp); overflow {
-		return 0, ErrGasUintOverflow
-	}
-
```

Comment:
For `gasCall` this chunk of logic is moved to `opCall(...)`  since we have to deal wit this situation of `TouchAndChargeValueTransfer` in (deleted) L410 could fail to have enough gas (thus we don't have to add it to the witness).

For all the rest of `gasCallCode/StaticCall/DelegateCall` we keep the 1/64 logic where it already lives. So this (unfortunate-ish) change only applies to `CALL` really.